<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1179</width>
    <height>693</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QTabWidget" name="tasks">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>1311</width>
      <height>671</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>0</number>
    </property>
    <widget class="QWidget" name="main">
     <attribute name="title">
      <string>Основная</string>
     </attribute>
     <widget class="QPushButton" name="start0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="n0fd">
      <property name="geometry">
       <rect>
        <x>440</x>
        <y>10</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="clear0">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="table0">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>511</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="me0lb">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>63</y>
        <width>111</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="txt00lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>421</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>Для решения основной задачи использована сетка с числом разбиений по x:  n =</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt01lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>36</y>
        <width>511</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>При пересчете задачи с половинным шагом максимальная разность приближенных решений</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt02lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>65</y>
        <width>91</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>составила: ε2 = </string>
      </property>
     </widget>
     <widget class="QLabel" name="txt03lb">
      <property name="geometry">
       <rect>
        <x>220</x>
        <y>65</y>
        <width>141</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>и соответствует узлу: x =</string>
      </property>
     </widget>
     <widget class="QLabel" name="mex0lb">
      <property name="geometry">
       <rect>
        <x>370</x>
        <y>66</y>
        <width>111</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graph0" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>210</y>
        <width>611</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="grapherr0" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>611</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="test1">
     <attribute name="title">
      <string>Тестовая №1</string>
     </attribute>
     <widget class="QTableWidget" name="table1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>511</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="start1">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear1">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="n1fd">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="me1lb">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="mex1lb">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>81</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="txt10lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Для решения тестовой задачи использована сетка с числом разбиений по x:  n =</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt11lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>36</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Требуемая точность решения тестовой задачи ε = 0.5e-6, тестовая задача решена с точностью</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt12lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ε1 = </string>
      </property>
     </widget>
     <widget class="QLabel" name="txt13lb">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> , максимальное отклонение точного и приближенного решений наблюдается </string>
      </property>
     </widget>
     <widget class="QLabel" name="txt14lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>в точке: x = </string>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graph1" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>210</y>
        <width>611</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="grapherr1" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>611</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="test2">
     <attribute name="title">
      <string>Тестовая №2</string>
     </attribute>
     <widget class="QTableWidget" name="table2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>150</y>
        <width>511</width>
        <height>461</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="start2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Старт</string>
      </property>
     </widget>
     <widget class="QPushButton" name="clear2">
      <property name="geometry">
       <rect>
        <x>100</x>
        <y>110</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Очистить</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="n2fd">
      <property name="geometry">
       <rect>
        <x>430</x>
        <y>10</y>
        <width>91</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
     <widget class="QLabel" name="txt23lb">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>60</y>
        <width>431</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string> , максимальное отклонение точного и приближенного решений наблюдается </string>
      </property>
     </widget>
     <widget class="QLabel" name="txt20lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>11</y>
        <width>421</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string>Для решения тестовой задачи использована сетка с числом разбиений по x:  n =</string>
      </property>
     </widget>
     <widget class="QLabel" name="txt24lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>80</y>
        <width>61</width>
        <height>21</height>
       </rect>
      </property>
      <property name="text">
       <string>в точке: x = </string>
      </property>
     </widget>
     <widget class="QLabel" name="txt21lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>36</y>
        <width>521</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Требуемая точность решения тестовой задачи ε = 0.5e-6, тестовая задача решена с точностью</string>
      </property>
     </widget>
     <widget class="QLabel" name="mex2lb">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>81</y>
        <width>81</width>
        <height>20</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QLabel" name="txt22lb">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>60</y>
        <width>31</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>ε1 = </string>
      </property>
     </widget>
     <widget class="QLabel" name="me2lb">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>60</y>
        <width>81</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QCustomPlot" name="graph2" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>210</y>
        <width>611</width>
        <height>391</height>
       </rect>
      </property>
     </widget>
     <widget class="QCustomPlot" name="grapherr2" native="true">
      <property name="geometry">
       <rect>
        <x>540</x>
        <y>10</y>
        <width>611</width>
        <height>191</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1179</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="geometry">
     <rect>
      <x>342</x>
      <y>100</y>
      <width>183</width>
      <height>72</height>
     </rect>
    </property>
    <property name="title">
     <string>Меню</string>
    </property>
    <addaction name="exit"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="action">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>QCustomPlot</class>
   <extends>QWidget</extends>
   <header>qcustomplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
